dallinger/command_line/docker_ssh.py

```diff
@@ -145,6 +147,28 @@ def prepare_server(host, user):
         print("Docker compose already installed")
 
 
+def copy_docker_config(host, user):
+    executor = Executor(host, user)
```

Comment:
Where does this name (`Executor`) come from?

Reply:
It's a local utility class to execute commands remotely using Paramiko and get their output in a variable.